Sources/TensorFlow/Operators/LinearAlgebra.swift

```diff
@@ -266,3 +266,122 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
         return (s, u, v)
     }
 }
+
+
+// MARK: Solvers
+
+
+/// Solves optionally batched systems of linear equations with upper or lower triangular matrices by backsubstitution.
+/// Returns solution to the system `A x = b`. Shape of return matches `b`.
+///
+/// - Parameters:
+///     - matrix: A batched matrix tensor.
+///     - rhs: A batched vector tensor.
```

Comment:
Please add parameter doc comments for all parameters: missing for `lower` and `adjoint`.

Reply:
The `lower` and `adjoint` parameter doc comments are missing a description. Please don't neglect those.

Reply:
done